UnitySDK/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -67,14 +81,14 @@ public override void CollectObservations()
         float rayDistance = 20f;
         float[] rayAngles = { 0f, 45f, 90f, 135f, 180f, 110f, 70f };
         string[] detectableObjects;
-        if (team == Team.red)
+        if (team == Team.Red)
         {
-            detectableObjects = new string[] { "ball", "redGoal", "blueGoal",
+            detectableObjects = new[] { "ball", "redGoal", "blueGoal",
```

Comment:
Why did you change that ? It makes it less explicit.

Reply:
The code is more concise. The extra string is redundant, because the contents are clearly already strings.